sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -692,7 +706,14 @@ case class ToUnixTimestamp(
  * second parameter.
  */
 @ExpressionDescription(
-  usage = "_FUNC_([expr[, pattern]]) - Returns the UNIX timestamp of current or specified time.",
+  usage = "_FUNC_([timeExp[, format]]) - Returns the UNIX timestamp of current or specified time.",
+  arguments = """
+    Arguments:
+      * timeExp - A date/timestamp or String. Default value is current time
```

Comment:
`String` -> `string`.

Reply:
Also, `Default value is current time`?

Reply:
As per usage, `timeExp` can be empty and if so, it'll default to the current time. This I believe still makes sense to be here

Reply:
Re-phrased this so it's a bit more clear